test/functional/api/cas/casadm.py

```diff
@@ -42,6 +42,36 @@ def start_cache(cache_dev: Device, cache_mode: CacheMode = None,
     return Cache(cache_dev)
 
 
+def start_standby_cache(cache_dev: Device, cache_line_size: CacheLineSize,
+                        cache_id: int, shortcut: bool = False,
+                        kernel_params: KernelParameters = KernelParameters()):
+    if kernel_params != KernelParameters.read_current_settings():
+        reload_kernel_module("cas_cache", kernel_params.get_parameter_dictionary())
+
+    output = TestRun.executor.run(start_cmd(
+        cache_dev=cache_dev.path, cache_line_size=cache_line_size, cache_id=cache_id,
```

Comment:
we should pass strings as arguments to start_cmd, that is we have to "stringify" cache_line_size and cache_id